task13/grid.css

```diff
@@ -0,0 +1,95 @@
+.wrap{
+    display:grid;
+    grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+    grid-template-rows: 320px 320px 320px;
+    grid-auto-rows: 320px;
+    grid-gap: 10px;
+    grid-auto-flow: dense;
+}
+
+.big-row{
```

Comment:
It is a modificator. And may be better to name this as cart-menu_big-row

Reply:
fixed class="card-menu card-menu_big-row"